recipe/build.sh

```diff
@@ -32,6 +32,9 @@ cmake -DBUILD_MCNP5=OFF \
       -DBUILD_PIC=OFF \
       -DBUILD_RPATH=ON \
       -DDOUBLE_DOWN=OFF \
+      -DCMAKE_POLICY_DEFAULT_CMP0074=NEW \
+      -DHDF5_ROOT="${PREFIX}" \
+      -DHDF5_FIND_DEBUG=ON \
```

Comment:
Does this have a continuing purpose? Or was it transient?

Reply:
Just debugging, although the fix I've proposed here didn't work as intended so I still have some digging to do :disappointed: